src/storage/types.rs

```diff
@@ -121,6 +122,22 @@ pub struct PrewriteResult {
     pub one_pc_commit_ts: TimeStamp,
 }
 
+#[derive(Clone)]
```

Comment:
why not add `Clone` to `cfg_attr(test...)` ?

Reply:
It's not supposed only to be used in tests.
But it's actually totally unused yet. I'll remove it.